core/local/steps/win_detect_move.js

```diff
@@ -71,7 +73,7 @@ function forget(
 
 function previousPaths(deletedPath, unmergedRenamedEvents) {
   const { previous: previousPaths } = unmergedRenamedEvents.reduceRight(
-    /* ::<{ previous: string[], current: string }> */ (paths, renamedEvent) => {
+    (paths, renamedEvent) => {
```

Comment:
I believe `/* ::`  with a space between `/*` and `::` is actually ignored by flow, so it is not supposed to actually check anything :see_no_evil: 
(I didn't want to figure out what should be the actual type annotation since I already had 100+ errors to fix)

Reply:
Hum from what I've seen, this is not ignored (I just tried to replace the type of `current` by `number` and Flow complained).

Reply:
Also looks like it completely breaks the eslint parser :man_facepalming: